public/js/utils/debug.js

```diff
@@ -1,11 +1,10 @@
 const { isDevelopment } = require("../feature");
+const flag = require("../test/test-flag");
```

Comment:
could we do the same thing with the `defineGlobal`? I'd prefer not introducing a new system just for this